app/src/main/AndroidManifest.xml

```diff
@@ -13,10 +13,13 @@
         <activity android:name="ch.sheremet.dasfoo.rover.android.client.MainActivity">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
-
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity
+            android:name=".menu.SettingsActivity"
```

Comment:
This looks crippled (starting with `.`), but I don't know how it works so feel free to ignore this comment.